frontend/public/components/cluster-service-broker.tsx

```diff
@@ -71,15 +71,15 @@ const ClusterServiceBrokerDetails: React.SFC<ClusterServiceBrokerDetailsProps> =
   </React.Fragment>;
 };
 
+const ServiceClassTabPage = ({obj}) => <ClusterServiceClassPage showTitle={false} fieldSelector={`spec.clusterServiceBrokerName=${obj.metadata.name}`} />;
 export const ClusterServiceBrokerDetailsPage: React.SFC<ClusterServiceBrokerDetailsPageProps> = props => <DetailsPage
   {...props}
   kind={ClusterServiceBrokerReference}
   menuActions={menuActions}
   pages={[
     navFactory.details(detailsPage(ClusterServiceBrokerDetails)),
     navFactory.editYaml(),
-    navFactory.clusterServiceClasses(({obj}) => <ClusterServiceClassPage showTitle={false}
```

Comment:
This created a new component on every render, which caused the browser to lock up. (The component updates the URL on mount when a filter is active.)